src/components/Dropdown/components/MultiValueContainer/MultiValueContainer.jsx

```diff
@@ -52,7 +60,7 @@ export default function Container({ children, selectProps, ...otherProps }) {
             disabled={isDisabled}
             id={option.value}
             label={option.label}
-            onDelete={onSelectedDelete}
+            onDelete={() => onDelete(option)}
```

Comment:
shouldn't it simply be
`onDelete={onDelete}`?

the `useCallback should have handled the callback

Reply:
Where would the `option` param come from then?

Reply:
I see. so in that case you should pass the `onDelete` callback to the dependency array of the `useCallback` one

Reply:
Fixed